docs/Getting-Started-frontend.md

```diff
@@ -240,23 +240,24 @@ The instructions below are for *NIX systems like OS X and Linux.
 
 Pre-reqs:
 
-1. Install [node.js](https://nodejs.org/) if you haven't already. On a Mac the recommended way to install Node is via [Homebrew](http://brew.sh/) via `brew install node`
-2. Install the [`grunt`](http://gruntjs.com/) command line tool via: `npm install -g grunt-cli`
+1. Install [node.js](https://nodejs.org/) if you haven't already. On a Mac the recommended way to install Node is via [Homebrew](https://brew.sh/) via: `brew install node`
+2. Install the [`grunt`](https://gruntjs.com/) command line tool via: `npm install -g grunt-cli`
+3. Install [`Yarn`](https://yarnpkg.com/en/docs/install).
 
-Once you have `npm` and `grunt`, clone this repo using `git` and within the `group-income-simple` directory run these two commands:
+Once you have `npm`, `grunt`, and `yarn`, clone this repo using `git`, and within the `group-income-simple` directory run these two commands:
 
 ```
-npm install
+yarn install
 grunt dev
 ```
 
 If all went well you should be able to visit: [http://localhost:8000](http://localhost:8000)
 
 ----
 
-#### What does `npm install` do?
+#### What does `yarn install` do?
 
-It installs the dependencies that this project relies on and places them into the `node_modules/` folder. We've setup grunt to verify you always have the latest dependencies installed, but you do need to run this command at least once.
+It installs the dependencies that this project relies on and places them into the `node_modules/` folder. The list of dependencies is retrieved from the `package.json` file, and stored in the `yarn.lock` file. We've set up `grunt` to verify you always have the latest dependencies installed, but you do need to run this command at least once.
```

Comment:
> The list of dependencies is retrieved from the `package.json` file, and stored in the `yarn.lock` file.

This is unclear. It sounds like it's just copying something for no reason. If you mention this, mention the reason for the yarn.lock file, or not all.